.github/workflows/release.yml

```diff
@@ -0,0 +1,66 @@
+on:
+  push:
+    tags:
+      - 'v*' # Push events to matching v*, i.e. v1.0, v20.15.10
+name: Release workflow
+jobs:
+  build-phar:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+
+      - name: Setup PHP
+        uses: shivammathur/setup-php@master
+        with:
+          php-version: 7.4
```

Comment:
shouldn't this be 7.3 as that's the min supported version ? Otherwise, the phar could contain incompatible packages